### PR TITLE
Emitting custom CEL from usersharedlines context

### PR DESCRIPTION
**Pull request by DrPyser**
https://wazo-dev.atlassian.net/browse/WAZO-3292

### CODE REVIEW

File: dialplan/asterisk/extensions_lib_user.conf [hunk 1]
```diff
@@ -239,7 +239,9 @@ same  =      n,Hangup()
 
 [usersharedlines]
 ; exten should be a user UUID
-exten = _[0-9a-f].,1,AGI(agi://${XIVO_AGID_IP}/get_user_interfaces,${EXTEN})
+exten = _[0-9a-f].,1,NoOp(Ringing lines of user ${EXTEN})
+same  = n,CELGenUserEvent(WAZO_ORIGINATE_ALL_LINES,user_uuid:${EXTEN},tenant_uuid:${WAZO_TENANT_UUID})
```
Comment (xxbillouxx) [inline dialplan/asterisk/extensions_lib_user.conf:243]: I don't know xivo-config... the `EXTEN` is the user uuid ?

**Review by xxbillouxx — COMMENTED** [review]
I have a small question :-)

File: dialplan/asterisk/extensions_lib_user.conf (hunk 1 above)
Comment (DrPyser) [reply dialplan/asterisk/extensions_lib_user.conf:243]: Yes, as indicated by the comment at line 241. When calld originates a call(from an API request) ringing all lines of a user for the user to choose which line to use, the originated call is forwarded to this dialplan context, with the extension being the user uuid(`{user_uuid}@usersharedlines`).

**Review by xxbillouxx — APPROVED** [review]
Thank you :-)

**Pull request merged**